api/v1/configurationpolicy_types.go

```diff
@@ -292,6 +293,23 @@ type ConfigurationPolicySpec struct {
 	ObjectTemplatesRaw string `json:"object-templates-raw,omitempty"`
 }
 
+// CustomMessage configures the compliance messages emitted by the configuration policy, to use one
+// of the specified Go templates based on the current compliance. The data passed to the templates
+// include a '.DefaultMessage' string which matches the message that would be emitted if no custom
+// template was defined, and a '.Policy' object which contains the full current state of the policy.
+// If the policy is using Kubernetes API watches (default but can be configured with
+// EvaluationInterval), and the object exists, then the full state of each related object will be
+// available at '.Policy.status.relatedObjects[*].object'. Otherwise, only the identifier
+// information will be available there.
```

Comment:
(Optional) Use backticks (since descriptions are Markdown) and add the word "variable" for those not as familiar with templates.
```suggestion
// CustomMessage configures the compliance messages emitted by the configuration policy, to use one
// of the specified Go templates based on the current compliance. The data passed to the templates
// include a `.DefaultMessage` string variable which matches the message that would be emitted if no custom
// template was defined, and a `.Policy` object variable which contains the full current state of the policy.
// If the policy is using Kubernetes API watches (default but can be configured with
// EvaluationInterval), and the object exists, then the full state of each related object will be
// available at `.Policy.status.relatedObjects[*].object`. Otherwise, only the identifier
// information will be available there.
```